src/analyzer/protocol/smtp/SMTP.cc

```diff
@@ -857,6 +858,15 @@ bool SMTP_Analyzer::ProcessBdatArg(int arg_len, const char* arg, bool orig) {
         return false;
     }
 
+    // The ContentLine analyzer only supports int64_t, but BDAT could deal
+    // with uint64_t sized chunks. Weird if the chunk size is larger and
+    // do not configure the ContentLine analyzer for plain delivery.
+    if ( chunk_size > std::numeric_limits<int64_t>::max() ) {
```

Comment:
Could this cause us to drop valid packets? Should we open an issue to investigate fixing the ContentLine analyzer?

Reply:
I think it's fine. 2**63 is is 8192 petabytes - here it's definitely weird and suspect immensely huge in other places, too. 
```
In [8]: 2**63/(1024*1024*1024*1024*1024)
Out[8]: 8192.0
```